spring-cloud-gcp-storage/pom.xml

```diff
@@ -10,15 +10,21 @@
 	</parent>
 
 	<artifactId>spring-cloud-gcp-storage</artifactId>
-
+	<properties>
+		<main.basedir>${basedir}/../..</main.basedir>
+	</properties>
 	<dependencies>
 		<dependency>
-			<groupId>com.google.cloud</groupId>
-			<artifactId>google-cloud</artifactId>
+			<groupId>org.springframework</groupId>
+			<artifactId>spring-context</artifactId>
 		</dependency>
 		<dependency>
-			<groupId>org.springframework.cloud</groupId>
-			<artifactId>spring-cloud-gcp-core</artifactId>
+			<groupId>commons-logging</groupId>
+			<artifactId>commons-logging</artifactId>
```

Comment:
Hm. OK. I guess Boot excludes it somehow from the `spring-core`: https://github.com/spring-projects/spring-boot/blob/master/spring-boot-starters/pom.xml#L85

OK, never mind.
Thank you for your patience!